### PR TITLE
refactor(es6): Upgrade Entrypoint to es6

### PR DESCRIPTION
**Pull request by TheLarkInn**
**What kind of change does this PR introduce?**
Refactor

**Did you add tests for your changes?**
Existing Tests are Passing 🍾 

**If relevant, link to documentation update:**
N/A 

**Summary**
Upgrade Entrypoint to es6 😍 

**Does this PR introduce a breaking change?**
Nopers!

### CODE REVIEW

File: lib/Entrypoint.js [hunk 1]
```diff
@@ -2,37 +2,42 @@
...
+	getFiles() {
+		let files = [];
 
-	for(var chunkIdx = 0; chunkIdx < this.chunks.length; chunkIdx++) {
-		for(var fileIdx = 0; fileIdx < this.chunks[chunkIdx].files.length; fileIdx++) {
-			if(files.indexOf(this.chunks[chunkIdx].files[fileIdx]) === -1) {
-				files.push(this.chunks[chunkIdx].files[fileIdx]);
+		for(let chunkIdx = 0; chunkIdx < this.chunks.length; chunkIdx++) {
+			for(let fileIdx = 0; fileIdx < this.chunks[chunkIdx].files.length; fileIdx++) {
+				if(files.indexOf(this.chunks[chunkIdx].files[fileIdx]) === -1) {
```
Comment (abouthiroppy) [inline lib/Entrypoint.js]: I think it is better to use `includes`.

Comment (TheLarkInn) [inline lib/Entrypoint.js]: Is that available in node v4. I'd have to check node.green to make sure

Comment (abouthiroppy) [inline lib/Entrypoint.js]: Sorry, `Array.prototype.includes` could not use at v4.7. 
`String.prototype.includes` can be used at 4.7.

Comment (TheLarkInn) [inline lib/Entrypoint.js]: No probs, thank you for double checking.

**Pull request merged**